users/views.py

```diff
@@ -31,3 +26,33 @@ class PaymentListAPIView(generics.ListAPIView):
         "paid_course",
         "payment_method",
     )
+
+
+class UserCreateAPIView(generics.CreateAPIView):
+    serializer_class = UserSerializer
+    queryset = User.objects.all()
+    permission_classes = (AllowAny,)
+
+    def perform_create(self, serializer):
+        user = serializer.save(is_active=True)
```

Comment:
Пользователь всегда создается `is_active`, дефолтное поведение, поэтому можно так:
```suggestion
        user = serializer.save()
```
А вот пароль, действительно лучше отдельным запросом поменять, хотя можно использовать `make_password` (не критично)